assets/styles/global.scss

```diff
@@ -51,10 +51,34 @@ h1 {
   margin-bottom: 2rem;
 }
 
+a {
+  color: inherit;
+
+  &:hover {
+    color: inherit;
+    text-decoration: none;
+  }
+}
+
 a:hover {
   cursor: pointer;
 }
 
+a.link {
+  font-weight: 400;
+  color: $udb-blue;
+
+  &:hover {
+    text-decoration: underline;
+  }
+}
+
+.panel {
```

Comment:
Feels like something that should go in a separate `panel` component, which would also fit the idea to create a component library and use Storybook.

Reply:
Will be implemented in next PR